src/components/DayPickerRangeController.jsx

```diff
@@ -89,6 +90,7 @@ const defaultProps = {
 
   renderDay: null,
   renderCalendarInfo: null,
+  calculateToday: true,
```

Comment:
ideally boolean props always default to `false`, such that absence and `false` are the same - the main exception is when the default is intended to be flipped later in a semver-major bump. If that's not the case here, could this be renamed so that it can default to `false`?